cmd/lncli/commands.go

```diff
@@ -84,6 +84,37 @@ func actionDecorator(f func(*cli.Context) error) func(*cli.Context) error {
 	}
 }
 
+// calculateFeeLimit takes ctx and the amount for the payment in satoshis.
+// It checks the fee_limit and fee_limit_percent flags and determines the
+// the feeLimit based on those values. If both flags are set, calculateFeeLimit
+// sets feeLimit to the Minumum value between them. If neither flag is set,
+// feeLimit is set to the maximum int64 value as to avoid enforcing a limit.
```

Comment:
Maybe there _should_ be a default limit, for safety? Paying fees that are a huge multiple of the amount transferred might be necessary and desirable in some extreme circumstances, but maybe the user should be protected from doing it unintentionally.

Reply:
Ok, should this limit be an absolute number or proportional to transaction amount? 

Off the top of my head, I'm thinking limit should be equal to, or some small multiple (2x or 3x) of the transaction amount. I say this because although there are circumstances in which someone might want to pay fees equal to or larger than the transaction amount, it seems reasonable to make users explicitly declare they want to go forward with the transaction.

Open to implementing this according to whichever rules you guys suggest though.